pages/index.js

```diff
@@ -1,7 +1,13 @@
+import LinkButton from "@/components/LinkButton";
+import Headline from "@/components/Headline";
+import ImageTextModule from "@/components/ImageTextModule";
+
 export default function HomePage() {
   return (
     <div>
-      <h1>Home Page</h1>
+      <ImageTextModule />
+      <Headline name="Find your dog" />
+      <LinkButton fullwidth={true} href="/quiz-page" name="Start the quiz!" />
```

Comment:
```suggestion
      <LinkButton fullwidth={true} href="/quiz-page">Start the quiz!</LinkButton>
```
This is the adjustment due to the recommendation in the `LinkButton` component.